types/smooth-scroll/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for smooth-scroll 14.2
+// Type definitions for smooth-scroll 15.0
```

Comment:
Shouldn't it be the latest version `16.1.0`?

Reply:
My mistake. Thanks!